cloudman/helmsman/helm/client.py

```diff
@@ -69,6 +43,10 @@ def repositories(self):
     def charts(self):
         return self._chart_svc
 
+    @property
+    def namespaces(self):
```

Comment:
I think we should have a seperate, standalone `KubernetesClient`? that wraps kubectl, just like the helmclient wraps helm?  In future, I'd imagine it would become more comprehensive, wrapping more functionality of kubectl as and when we need them.

Reply:
I can bring back the separate client